libs/output-mapping/src/Storage/TableStructureValidator.php

```diff
@@ -56,16 +57,13 @@ public function validateTable(string $tableId, array $schemaColumns): TableChang
             );
             array_walk($missingColumns, fn($column) => $tableChangesStore->addMissingColumn($column));
 
-            $this->validatePrimaryKeys(
-                $table['definition']['primaryKeysNames'],
-                array_map(
-                    fn(MappingFromConfigurationSchemaColumn $column) => $column->getName(),
-                    array_filter(
-                        $schemaColumns,
-                        fn(MappingFromConfigurationSchemaColumn $column) => $column->isPrimaryKey(),
-                    ),
-                ),
-            );
+            $primaryKey = new MappingFromConfigurationSchemaPrimaryKey();
+            foreach ($schemaColumns as $schemaColumn) {
+                if ($schemaColumn->isPrimaryKey()) {
+                    $primaryKey->addPrimaryKeyColumn($schemaColumn);
+                }
+            }
```

Comment:
K tomuhle se musí dodělat testy. `TableStructureValidatorTest` bude chtít ještě péči. Ty typed testy vevnitř se pouští nad non-typed tabulkou.

Reply:
na callu jsme se domluvili že to tak necháme